frontend/src/host_orchestrator/orchestrator/instancemanager.go

```diff
@@ -346,8 +346,8 @@ type startCVDParams struct {
 	BootloaderDir string
 }
 
-func (h *startCVDHandler) Start(p startCVDParams) error {
-	args := []string{groupNameArg, "start", daemonArg, reportAnonymousUsageStatsArg}
+func (h *startCVDHandler) Create(p startCVDParams) error {
```

Comment:
yes, but that causes a name conflict with the request handler somewhere else. I'll do the rename in another PR as it could get big and involve other things not related to this change.